openapi.yaml

```diff
@@ -475,19 +475,19 @@ paths:
               properties:
                 app_name:
                   type: string
-                  description: Required unless using `app_id`. The name of the AI Hub app to run.
+                  description: Required unless using `app_id`. The name of the automation app to run.
```

Comment:
Hmm... Do we need to say "automation app" every time? I agree with using "automation app" in the API description (line 452), but I think after first use, we can just say "app". It gets redundant to repeat "automation app" so much. 

+1 to removing the "AI Hub" here though.